app/assets/javascripts/admin/addon/mixins/setting-component.js

```diff
@@ -191,120 +179,37 @@ export default Mixin.create({
     }
   }),
 
-  disableSaveButton: computed("isSaving", "validationMessage", function () {
-    return !!this.validationMessage || this.isSaving;
```

Comment:
I think this was technically incorrect. You'd enter a value, a request is sent, then you are shown an error message. Even if you edit the value the save button remained disabled. You'd have to clear the whole thing with the reset button and start over.